drivers/bus/acpi/acpica/include/platform/acwin64.h

```diff
@@ -99,6 +100,46 @@
 
 #endif
 
+#else /* __REACTOS__ */
+
+#ifdef ACPI_APPLICATION
+#define ACPI_FLUSH_CPU_CACHE()
+#else
+#define ACPI_FLUSH_CPU_CACHE()  __wbinvd()
+#endif
+
+#define ACPI_ACQUIRE_GLOBAL_LOCK(FacsPtr, Acq) \
```

Comment:
Wouldn't it be interesting to use instead a FORCEINLINE FASTCALL function? (and for the release_global_lock too)

Reply:
The way the Acq parameter is used doesn't allow that.